File: PAT/PAT_B1011.cpp
// PAT B1011

#include <cstdio>
#include <iostream>
using namespace std;

int main() {
   int caseNum;
   long long a, b, c;

   cin >> caseNum;
   for(int i = 0; i < caseNum; i++) {
       cin >> a >> b >> c;
       if(a + b > c) {
           printf("Case #%d: true\n", i+1);
       }
       else {
           printf("Case #%d: false\n", i+1);
       }
   }

    return 0;
}

File: PAT/PAT_B1011.py
# PAT B1011

case_num = int(input())
for i in range(0, case_num):
    entry = input()
    numbers = entry.split(' ')
    a = int(numbers[0])
    b = int(numbers[1])
    c = int(numbers[2])
    print('Case #' + str(i + 1) + ': ', end='')
    if a + b > c:
        print('false')
    else:
        print('true')
